modules/solid_mechanics/include/materials/SolidModel.h

```diff
@@ -160,6 +161,9 @@ class SolidModel : public DerivativeMaterialInterface<Material>
   MaterialProperty<ColumnMajorMatrix> * _Eshelby_tensor;
   MaterialProperty<RealVectorValue> * _J_thermal_term_vec;
 
+  //This is used in calculation of the interaction integral
+  MaterialProperty<Real> * _thermal_expansion_coeff;
```

Comment:
There's a potential for confusion of this with the user-supplied alpha. I think I'd change the name here to make it a little more clear what this is for.  Maybe `_current_instantaneous_thermal_expansion_coef`?  It's wordy, but that would discourage people from misusing it.